### DISCUSSION TITLE
Fail to run exmaple `todo`

### DISCUSSION THREAD

**Discussion by snylonue** [hecrj/iced#976] [Q&A] [answered]
I cloned `master` branch and ran `cargo run --package todos`
Then a white window brought up and exited with output in the terminal below
```
    Finished dev [unoptimized + debuginfo] target(s) in 0.57s
     Running `target\debug\todos.exe`
error: Invalid record (Producer: 'LLVM3.8.0' Reader: 'LLVM 3.8.0')
error: process didn't exit successfully: `target\debug\todos.exe` (exit code: 0xc0000005, STATUS_ACCES
S_VIOLATION)
[1]    423 segmentation fault  cargo run --package todos
```

**Comment by inspiretk** [comment]
I just tested it on windows 10, it worked for me. I downloaded iced, then ran your command, and it worked.

**Comment by snylonue** [comment]
> 
> 
> I just tested it on windows 10, it worked for me. I downloaded iced, then ran your command, and it worked.

I'm on Windows 10 too. 
I just found that `0.3` worked for me but `master` branch didn't.

There is a pdb file that might help but it's too big to be uploaded

**Comment by 13r0ck** [comment, marked as answer]
Your error mentions LLVM, which is the compiler for Rust. Your error is probably with your Rust install

**Reply by snylonue** [reply]
It works after I re-installed rust

**Comment by ddki** [comment]
我也遇到了这个问题，更新下显卡驱动就好了

**Reply by savire** [reply]
Lol this is true after installing latest graphics driver from Nvidia its working. That is weird.
